vulcan/lib/client/jsx/spec/fixtures/sample_cwl.yaml

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
```

Comment:
We probably don't actually need the bash line, I don't think we'll want to have this shebang in our files in general.

Reply:
but eh nevermind don't mind me